### PR TITLE
from renku aqs to renku graphvis

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by burnout87** [comment]
I suggest to have a first round of review can be done, and continue from there.

This contains only the two main CLI functionalities contained in https://github.com/oda-hub/renku-aqs , and jupyter server extension for renkulab

File: README.md [hunk 1]
```diff
@@ -1 +1,130 @@
-# renku-graph-vis
\ No newline at end of file
+# renku-graph-vis plugin
+
+## Graphical visualization of the graph
+Starting from the knowledge graph extracted from the renku project, this is queried to retrieve the needed information,
```
Comment (volodymyrss) [inline README.md:4]: I suggest to feed it through GPT or @ferrigno. Or I will adapt it a bit later.

Comment (volodymyrss) [reply README.md:4]: Or I can keep making concrete suggestions for improvement, if you prefer.

**Comment by ferrigno** [comment]
The README at https://github.com/oda-hub/renku-aqs contains a lot of rather obscure jargon: it does not give much context and does not really explain why one should use this feature. 

Above all, see issue #65 I cannot even try it...

**Comment by volodymyrss** [comment]
> The README at https://github.com/oda-hub/renku-aqs contains a lot of rather obscure jargon: it does not give much context and does not really explain why one should use this feature.
> 
> Above all, see issue #65 I cannot even try it...

Although this readme should be improved quite a bit, it would remain more involved than a simplified guide just for workflow developers which is being added in here https://github.com/oda-hub/hugo-odahub/pull/56 - by the way, I tagged you there, could you check that user-facing doc?

Finally, there is split ongoing in this project, it becomes two plugins.

**Comment by ferrigno** [comment]
Thanks for pointing to the right doc to look at. That doc is more comprehensible.  I see different installation instructions there

File: README.md (hunk 1 above)
Comment (burnout87) [reply README.md]: I did a first round of adaption using ChatGPT. In the meantime you can comment on a more overall level, is some thing missing? something should be removed/added?

File: README.md [hunk 2]
```diff
@@ -1 +1,141 @@
-# renku-graph-vis
\ No newline at end of file
+# renku-graph-vis plugin
+
+This Renku plugin offers a visual representation of the project Knowledge Graph. Users can access the visualization through a CLI command, as well as during an interactive Renku session. The plugin allows users to have a live overview of the project's development through the Knowledge Graph.
+
+Additionally, the plugin provides an option to enrich the graph with an ontology. By doing so, the plugin can identify specific types of individual components, thus enhancing the understanding of those. This enrichment allows users to gain a better understanding of the interconnections within the Knowledge Graph.
+
+## Graphical visualization of the graph
```
Comment (volodymyrss) [inline README.md:7]:
```suggestion
## Visualization of the graph
```

**Pull request merged**